Pushing my theme files to Dark_Typora

AlexFlipnote opened the pull request:
Pushing my theme files to Dark_Typora for the first time to `typora/typora-theme-gallery`. All files have been tested by [this website](http://theme.typora.io/doc/Submit-A-Theme/).

AlexFlipnote committed:
Pushing my theme files to Dark_Typora

abnerlee commented:
Thanks,

it it is a `fork` of the Github theme, you could follow https://github.com/typora/typora-theme-gallery/pull/74/files to submit the theme as a fork of current theme

AlexFlipnote committed:
Moved dir after comment suggestion.

AlexFlipnote commented:
Moved to fork folder and changed category to `fork` as well.

Curious question, are forked themes able to be listed on the site or are they only getting a URL while only non-forks are listed on index (http://theme.typora.io/)

abnerlee commented:
> Curious question, are forked themes able to be listed on the site

Currently no, but we plan to add marks on themes that have `forks` by other users in first page in future.

BTW, please make a link on the original theme page (`_posts/theme/2016-06-14-Github.md `), like https://github.com/typora/typora-theme-gallery/pull/74/files#diff-5272fefdd529627c245fb47f05ea30d1R39
